TTHAnalysis/python/plotter/mcPlots.py

```diff
@@ -1187,6 +1187,8 @@ def addPlotMakerOptions(parser, addAlsoMCAnalysis=True):
     outname  = options.out if options.out else (args[2].replace(".txt","")+".root")
     if (not options.out) and options.printDir:
         outname = options.printDir + "/"+os.path.basename(args[2].replace(".txt","")+".root")
+    if '/pool/ciencias' in outname:
+        outname = outname.replace('/pool/ciencias/','/pool/cienciasrw/')
```

Comment:
I switched to setting an --env variable at runtime. This final solution should be fine also for central, as it is the same that has been adopted in susy-interface, no?  (if it is not, then central should review its way of dealing with people having other computing centers than cern :octopus:  )